security-privacy-questionnaire.md

```diff
@@ -76,10 +66,18 @@ No.
 
 ## 11. Does this specification allow an origin some measure of control over a user agent's native UI?
 
-Yes. This feature allows an origin (usually a bank) to associate an icon and a
-descriptive string with a public key credential. This icon and string are
-displayed to the user in the user agent's native UI when this credential is
-exercised via Payment Request API.
+Yes. This feature allows an origin (usually a bank) to provide a set of
+information that will be shown to the user in a native UI:
+
+* Payment instrument icon and string
+* Payment amount and currency
+* Intended payee
+
+This information is displayed to the user in the user agent's native UI when
+this credential is exercised via Payment Request API. It could be spoofed by
+the merchant, however the issuing bank that receives the assertion will be able
+to detect this and reject the payment (and take action against the merchant as
+appropriate).
```

Comment:
Do you want to also say: 

"It could be in part spoofed by the merchant, but since the UX is browser controlled, it is rendered atop the chrome (which cannot be done via JavaScript)" 

(or however you would say that.)

It might be worth mentioning even if it's not the primary mitigation observation.

Reply:
That is not relevant to the question, which is about site control over browser UX. The idea of a site spoofing browser UX (which does not involve controlling it!) is true for every browser UX in existence (we see it for notifications, for WebAuthn, for ...)

Reply:
So what do you mean when you say "spoofed by the merchant"?

Reply:
The merchant can input incorrect data into the SPC API. Possibly spoofing is the wrong word there.

Reply:
How about: "If the merchant provides incorrect or fraudulent data to the API for confirmation by the user, the Relying Party can detect this when validating the assertion and reject the transaction."

Reply:
SGTM, done.